### ISSUE TITLE
Text should have a color contrast ratio of 4.5:1 or greater with the background.

### ISSUE THREAD

**Issue by Jkrzy** [18F/samhsa-prototype#187]
A few text color/contrast issues: https://accessibility.18f.gov/color/

Tested w/ https://developer.paciellogroup.com/resources/contrastanalyser/

Grey text on white background is 4:1 contrast ratio.
Results page information is 1.5:1
<img width="418" alt="Screen Shot 2019-08-05 at 4 45 32 PM" src="https://user-images.githubusercontent.com/3485564/62494094-aa38a480-b7a0-11e9-9c4a-7c40d179f172.png">
<img width="523" alt="Screen Shot 2019-08-05 at 4 45 36 PM" src="https://user-images.githubusercontent.com/3485564/62494127-be7ca180-b7a0-11e9-9746-d811a98c2fc8.png">
<img width="344" alt="Screen Shot 2019-08-05 at 4 48 36 PM" src="https://user-images.githubusercontent.com/3485564/62494209-e8ce5f00-b7a0-11e9-92bb-317017dbe546.png">

_Originally posted by @Jkrzy in https://github.com/18F/samhsa-prototype/issues/184#issuecomment-518394273_

Contrast ratio: 4.03:1
![Screen Shot 2019-08-06 at 10 58 39 AM](https://user-images.githubusercontent.com/3485564/62551021-3b5b5a00-b839-11e9-82d0-019980f73882.png)

**Comment by Jkrzy** [comment]
Closing this w/ the merge of #193.

We're expecting additional design changes and will revisit this if necessary.
